packages/Datadog.Unity/Documentation/README.md

```diff
@@ -57,6 +57,32 @@ Before Datadog sends any data, we need to confirm the user’s Tracking Consent.
 DatadogSdk.Instance.SetTrackingConsent(TrackingConsent.Granted);
 ```
 
+## Using the Datadog SDK in C#
+
+If you want to make calls to the Datadog SDK from your project's scripts, you can use `DatadogSdk.Instance` to access the SDK's public interface. See [`DatadogSdk.cs`](../Runtime/DatadogSdk.cs) for more details.
+
+### Providing error information
+
+Several functions in the SDK's public interface allow you to specify the details of an error. For example, you might want to emit a log message when your code handles an exception:
+
+```csharp
+try
+{
+    DoTask();
+}
+catch (Exception e)
+{
+    Dictionary<string, object> attributes = null;
+    DatadogSdk.Instance.DefaultLogger.Warn("Task failed", attributes, e);
+}
+```
+
+In this example, the log message that appears in Datadog will show the full details of the exception, including the stack trace. If the exception occurred in native code generated by IL2CPP, Datadog will show the full native stack trace.
+
+Functions that allow you to provide error information use the [`ErrorInfo`](../Runtime/ErrorInfo.cs) type. You can pass an `Exception` anywhere that the API expects an `ErrorInfo`.
+
+You may also construct an `ErrorInfo` manually in order to provide the Datadog SDK with arbitrary error information. However, you should prefer passing `Exception` values where possible, since native IL2CPP stack traces can only be recovered from a C# `Exception`.
```

Comment:
```suggestion
You may also construct an `ErrorInfo` manually to provide the Datadog SDK with arbitrary error information. However, prefer passing `Exception` values where possible, since native IL2CPP stack traces can only be recovered from a C# `Exception`.
```